utils/motorcontroller/SparkMAXFactory.java

```diff
@@ -39,12 +39,19 @@ public static class Config {
     }
 
     public static final Config defaultConfig = new Config();
+    public static final Config defaultVelocityConfig = new Config();
+    public static final Config defaultPositionConfig = new Config();
+    public static final Config defaultLowUpdateRateConfig = new Config();
     public static final Config defaultFollowerConfig = new Config();
 
     static {
         defaultFollowerConfig.STATUS_FRAME_0_RATE_MS = 1000;
         defaultFollowerConfig.STATUS_FRAME_1_RATE_MS = 1000;
         defaultFollowerConfig.STATUS_FRAME_2_RATE_MS = 1000;
+        defaultVelocityConfig.STATUS_FRAME_1_RATE_MS = 60;
+        defaultPositionConfig.STATUS_FRAME_2_RATE_MS = 60;
+        defaultLowUpdateRateConfig.STATUS_FRAME_1_RATE_MS = 60;
+        defaultLowUpdateRateConfig.STATUS_FRAME_2_RATE_MS = 60;
     }
```

Comment:
there are now more frames exposed: https://codedocs.revrobotics.com/java/com/revrobotics/cansparklowlevel.periodicframe, not just 0-2

Reply:
They're defined here: https://docs.revrobotics.com/brushless/spark-max/control-interfaces#periodic-status-frames

Reply:
I just saw the conifg and assumed that because it was done for the follower we actually pushed those to the spark maxes. Let me fix that